ui/app/routes/vault/cluster/secrets/backend/secret-edit.js

```diff
@@ -330,9 +337,10 @@ export default Route.extend(UnloadModelRoute, {
       // and explicity ignore it here
       if (
         (mode !== 'show' && changedKeys.length && changedKeys[0] !== 'backend') ||
-        (mode !== 'show' && version && Object.keys(version.changedAttributes()).length)
+        (mode !== 'show' && version && version.hasDirtyAttributes)
```

Comment:
`changedAttributes` does not clear after save or rollback finishes, so we use `hasDirtyAttributes` instead to track model cleanliness